database/schema.sql

```diff
@@ -5,28 +5,16 @@ CREATE DATABASE yumpSF;
 USE yumpSF;
 
 CREATE TABLE restaurants (
-  id INTEGER AUTO_INCREMENT PRIMARY KEY,
-  name TEXT NOT NULL 
+  id INTEGER AUTO_INCREMENT NOT NULL PRIMARY KEY,
+  restaurant_name TEXT NOT NULL 
 );
 
 CREATE TABLE dishes (
-  id INTEGER AUTO_INCREMENT NOT NULL PRIMARY KEY,
+  dish_id INTEGER AUTO_INCREMENT NOT NULL PRIMARY KEY,
   restaurant_id INTEGER NOT NULL,
```

Comment:
I would consider making this restaurant_id a foreign key.

Reply:
Thanks for pointing that out.